plugins/headlamp-pod-counter/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think we shouldn't mix plugins with the `headlamp-plugin` pkg at the same level, and we're also not intending yet to ship any official plugins in the repo. Therefore, I'd rather have the structure as:
```
plugins/
  README.md
  headlamp-plugin/
  examples/
    pod-counter/
    ...
```

Reply:
Yeah, this is better. It's this way now.